### DISCUSSION TITLE
Best Friends Forever

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#384] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/best-friends-forever/

**Comment by angellisy** [comment]
SEU MONSTRO!!!

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [quarta-feira, 23 de março de 2022](<https://www.vish.blog.br/imagens/best-friends-forever/#comment_1076890403>):

> SEU MONSTRO!!!

[![null](<https://camo.githubusercontent.com/e499b3a71e84bb28a2f61e3920ee98be37d05aedd154351facc4512017432173/68747470733a2f2f692e706f7374696d672e63632f6a355458355968642f652d74612d65727261646f2d706f7272612e6a7067>)](<https://camo.githubusercontent.com/e499b3a71e84bb28a2f61e3920ee98be37d05aedd154351facc4512017432173/68747470733a2f2f692e706f7374696d672e63632f6a355458355968642f652d74612d65727261646f2d706f7272612e6a7067>)

**Comment by tali-zorah** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [quarta-feira, 23 de março de 2022](<https://www.vish.blog.br/imagens/best-friends-forever/#comment_1076891943>):

> [![null](<https://camo.githubusercontent.com/e499b3a71e84bb28a2f61e3920ee98be37d05aedd154351facc4512017432173/68747470733a2f2f692e706f7374696d672e63632f6a355458355968642f652d74612d65727261646f2d706f7272612e6a7067>)](<https://camo.githubusercontent.com/e499b3a71e84bb28a2f61e3920ee98be37d05aedd154351facc4512017432173/68747470733a2f2f692e706f7374696d672e63632f6a355458355968642f652d74612d65727261646f2d706f7272612e6a7067>)

*Keelah...* não dá pra discordar
